build.gradle

```diff
@@ -23,24 +25,58 @@ description = 'HiveMQ CE is a Java-based open source MQTT broker that fully supp
 
 mainClassName = 'com.hivemq.HiveMQServer'
 
-ext {
-    readableName = 'HiveMQ Community Edition'
-    url = 'https://www.hivemq.com/'
-    vendor = 'dc-square GmbH'
-    githubOrg = 'hivemq'
-    githubRepo = 'hivemq-community-edition'
-    githubUrl = 'https://github.com/' + githubOrg + '/' + githubRepo
-    scmConnection = 'scm:git:git://github.com/' + githubOrg + '/' + githubRepo + '.git'
-    scmDeveloperConnection = 'scm:git:ssh://git@github.com/' + githubOrg + '/' + githubRepo + '.git'
-    issuesUrl = githubUrl + '/issues'
-    licenseShortName = 'Apache-2.0'
-    licenseReadableName = 'The Apache License, Version 2.0'
-    licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+metadata {
+    readableName = "HiveMQ Community Edition"
+    organization {
+        name = "dc-square GmbH"
+        url = "https://www.hivemq.com/"
+    }
+    license {
+        apache2()
+    }
+    developers {
+        developer {
+            it.id = "cschaebe"
+            it.name = "Christoph Schaebel"
+            it.email = "christoph.schaebel@hivemq.com"
+        }
+        developer {
+            it.id = "lbrandl"
+            it.name = "Lukas Brandl"
+            it.email = "lukas.brandl@hivemq.com"
+        }
+        developer {
+            it.id = "flimpoeck"
+            it.name = "Florian Limpoeck"
+            it.email = "florian.limpoeck@hivemq.com"
+        }
+        developer {
+            it.id = "sauroter"
+            it.name = "Georg Held"
+            it.email = "georg.held@hivemq.com"
+        }
+        developer {
+            it.id = "SgtSilvio"
+            it.name = "Silvio Giebl"
+            it.email = "silvio.giebl@hivemq.com"
+        }
+    }
+    github {
+        org = "hivemq"
+        repo = "hivemq-community-edition"
+        issues()
+        pages()
```

Comment:
We do not have GitHub Pages for HiveMQ CE

Reply:
Didnt you create one?

Reply:
Noe I am confused

Reply:
We do not have "https://hivemq.github.io/hivemq-community-edition/"
We just have "https://hivemq.github.io/nightly-builds/" but this is something different

Reply:
ok